anybotics_anymal_c/README.md

```diff
@@ -34,6 +34,21 @@ description](https://github.com/ANYbotics/anymal_c_simple_description).
 11. Softened the contacts of the feet to approximate the effect of rubber.
 12. Added `scene.xml` which includes the robot, with a textured groundplane, skybox, and haze.
 
+## MJX
+
+The MuJoCo config in `anymal_c_mjx.xml` has the following edits made specifically for MJX:
```

Comment:
A version of the ANYmal C for use in [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html) is available in `scene_mjx.xml` with the following changes: